### DISCUSSION TITLE
It would be nice to give permissiontrust to already give the other permissions without having to give separately

### DISCUSSION THREAD

**Discussion by thiagogebrimm** [GriefPrevention/GriefPrevention#1682] [Ideas]
It would make it easier, a lot of people get confused because of it

**Comment by Jikoo** [comment]
Permission trust is a separate track of trust essentially, it does not grant any tier of permission on its own. I agree that it makes sense to require users to have at minimum access trust to get perm trust, but since it's not tied to an actual level of trust, granting one would be unexpected behavior.

If you want permissiontrust to forcibly grant trust, you can add an entry to your [commands.yml](https://bukkit.fandom.com/wiki/Commands.yml) to override it, running `griefprevention:permissiontrust $$1-` and `griefprevention:trust $$1-`.

**Reply by thiagogebrimm** [reply]
I'll try
